x-pack/distributions/internal/observabilitySRE/qa/acceptance/docker/logstash/pipeline/logstash-to-elasticsearch.conf

```diff
@@ -0,0 +1,28 @@
+input {
+  generator {
+    # Generate this message indefinitely to give ES container time to come online
+    count => -1
```

Comment:
in theory we shouldn't need to generate more than one, since the ES output is designed to retry its batch of events until all of them have been explicitly rejected by elasticseaerch (e.g., with a successful HTTP 2XX response from the bulk API containing individual rejections).